### ISSUE TITLE
Moto tests failing; can't find eu-south-1 region

### ISSUE THREAD

**Issue by jjbuck** [spulec/moto#3243]
This is sort of a follow up to https://github.com/spulec/moto/issues/2789. I was running into the problem reported in that issue, and followed the suggestion to upgrade to 1.3.15.dev550+

I've upgraded to moto-1.3.15.dev751, and now I'm running into an error related to what seems to be a region map or configuration that assumes the presence of the eu-south-1 region. The stack trace is reproduced below.

```
Traceback (most recent call last):
  File "/bin/test-package.py", line 5, in <module>
    import moto
  File "/build/lib/python2.7/site-packages/moto/__init__.py", line 3, in <module>
    from .acm import mock_acm  # noqa
  File "/build/lib/python2.7/site-packages/moto/acm/__init__.py", line 2, in <module>
    from .models import acm_backends
  File "/build/lib/python2.7/site-packages/moto/acm/models.py", line 7, in <module>
    from moto.ec2 import ec2_backends
  File "/build/lib/python2.7/site-packages/moto/ec2/__init__.py", line 2, in <module>
    from .models import ec2_backends
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 5188, in <module>
    region.name: EC2Backend(region.name) for region in RegionsAndZonesBackend.regions
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 5188, in <dictcomp>
    region.name: EC2Backend(region.name) for region in RegionsAndZonesBackend.regions
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 5101, in __init__
    super(EC2Backend, self).__init__()
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 858, in __init__
    super(InstanceBackend, self).__init__()
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 1140, in __init__
    super(TagBackend, self).__init__()
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 2386, in __init__
    super(EBSBackend, self).__init__()
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 1922, in __init__
    super(SecurityGroupBackend, self).__init__()
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 1368, in __init__
    self._load_amis()
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 1375, in _load_amis
    self.amis[ami_id] = Ami(self, **ami)
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 1326, in __init__
    volume = self.ec2_backend.create_volume(15, region_name)
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 2390, in create_volume
    zone = self.get_zone_by_name(zone_name)
  File "/build/lib/python2.7/site-packages/moto/ec2/models.py", line 1731, in get_zone_by_name
    for zone in self.zones[self.region_name]:
KeyError: u'eu-south-1'
:::: src: FAILED
BUILD FAILED 

```

I then upgrade my version of botocore to 1.17.45, but I'm still running into the same issue.

Please advise.

**Comment by bblommers** [comment]
Hi @jjbuck, the `eu-south-`-region was only made available as of moto >= 1.3.15.dev772.

Let me know if upgrading solves both your problems.

FYI: The region-issue happens whenever botocore adds a new region that is not yet recognized by moto. The workaround (until moto supports the region) is typically to downgrade to a version of botocore that doesn't have the new region yet.

**Comment by bblommers** [comment]
Going to close this due to a lack of response, but feel free to let me know if this is still an issue.
